addons/account_fleet/views/fleet_vehicle_log_services_views.xml

```diff
@@ -13,10 +13,17 @@
                         type="object"
                         class="oe_stat_button"
                         icon="fa-pencil-square-o"
-                        invisible="not account_move_line_id.move_id">
+                        invisible="not account_move_line_id.move_id or not account_move_line_id">
```

Comment:
a move line must have a move - so either condition will have the same effect, no?